tox.ini

```diff
@@ -25,6 +25,7 @@ deps =
 [testenv:lint]
 commands =
     poetry install -q
+    poetry update flake8
```

Comment:
flake8 should be replaced by its superset ruff.  I do not want to do that until the existing tests pass.  Poetry and CI is a PITA in my opinion because you ask it to upgrade stuff and it just refuses.

Reply:
```suggestion
```